From eb5ac7d864052ae0b22ecc3c8e8a46b54e04f46c Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:08 -0400
Subject: [redhat] kabi: Adding symbol __genl_register_family (net/netlink/genetlink.c)

Message-id: <e097c9688d90ace22fbe9bb815ce581ce125c040.1410510773.git.poros@redhat.com>
Patchwork-id: 95904
O-Subject: [PATCH RHEL7.1 008/233] kabi: Adding symbol __genl_register_family (net/netlink/genetlink.c)
Bugzilla: 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    Netlink API

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__genl_register_family
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__genl_register_family
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__genl_register_family

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__genl_register_family b/redhat/kabi/kabi-module/kabi_ppc64/__genl_register_family
new file mode 100644
index 0000000..89da041
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__genl_register_family
@@ -0,0 +1,2 @@
+#1-
+0x6b435680 __genl_register_family vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__genl_register_family b/redhat/kabi/kabi-module/kabi_s390x/__genl_register_family
new file mode 100644
index 0000000..22b531a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__genl_register_family
@@ -0,0 +1,2 @@
+#1-
+0xec3ecefb __genl_register_family vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__genl_register_family b/redhat/kabi/kabi-module/kabi_x86_64/__genl_register_family
new file mode 100644
index 0000000..9ec734e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__genl_register_family
@@ -0,0 +1,2 @@
+#1-
+0xcc1d296d __genl_register_family vmlinux EXPORT_SYMBOL
-- 
1.7.1